src/SDKs/Resource/Management.ResourceManager/Generated/DeploymentsOperations.cs

```diff
@@ -1841,7 +1841,7 @@ internal DeploymentsOperations(ResourceManagementClient client)
             string _requestContent = null;
             if(parameters != null)
             {
-                _requestContent = SafeJsonConvert.SerializeObject(parameters, Client.SerializationSettings);
+                _requestContent = Rest.Serialization.SafeJsonConvert.SerializeObject(parameters, Client.SerializationSettings);
```

Comment:
Please revert. We have our own [SerializeObject ](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Resource/Management.ResourceManager/Utility/SafeJsonConvertWrapper.cs) which needs to be used instead

Reply:
Done